### DISCUSSION TITLE
Exclude Videos

### DISCUSSION THREAD

**Discussion by loudmicro** [immich-app/immich#4238] [Q&A]
Hello everyone, I'm just intrested in backing up my photos, not the videos. Is there a way to exclude formats (MP4) or directlly exclude all videos?

Thanks in advance

**Comment by alextran1502** [comment]
In the mobile app, the album selection interface for backing up, you can double tap the video album/folder to exclude them from being uploded

**Reply by loudmicro** [reply]
Thanks for the quick answer, I thought about that too but my phone stores all the camera photos and videos on the same folder (/Camera).

**Comment by rosaLux161** [comment]
That would be really nice. As already said lot of camera apps save the videos in the same folder. Now if you have e.g. 15 10GB videos it is almost impossible to upload all pictures because when the video upload stocks it restarts again the next time and does not upload any pictures.
